utilities/bitwarden.py

```
@@ -23,19 +27,42 @@ def _run_bws_command(args: list[str]) -> Any:
+    Args:
+        access_token: Bitwarden access token
+        args: Command arguments to pass to bws
+
+    Returns:
+        Any: Parsed JSON response from bws CLI (always truthy for retry decorator)
+
+    Raises:
+        TimeoutExpiredError: If bws CLI repeatedly fails within the retry window
+    """
+    success, stdout, stderr = run_command(
         command=["bws", "--access-token", access_token] + args,
```

Ruff (0.15.10)

[warning] Consider `["bws", "--access-token", access_token, *args]` instead of concatenation

Replace with `["bws", "--access-token", access_token, *args]`

(RUF005)